dll/cpl/desk/lang/fr-FR.rc

```diff
@@ -80,23 +80,23 @@ FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL "", IDC_APPEARANCE_PREVIEW, "PreviewWndClass", WS_VISIBLE | WS_BORDER, 7, 7, 232, 120
     LTEXT "Si vous sélectionnez un paramètre de boutons et de fenêtres autre que ReactOS classique, celui-ci va remplacer les paramètres suivants exception faite de certains programmes plus anciens.", IDC_ADVAPPEARANCE_DISCLAIMER, 7, 133, 232, 30, BS_MULTILINE
-    LTEXT "Élément :", 8, 5, 160, 50, 9
+    LTEXT "Élém&ent :", 8, 5, 160, 50, 9
     COMBOBOX IDC_ADVAPPEARANCE_ELEMENT, 5, 170, 130, 90, CBS_DROPDOWNLIST |
              CBS_HASSTRINGS | CBS_SORT | WS_CHILD | WS_VSCROLL | WS_TABSTOP
     LTEXT "&Taille :", IDC_ADVAPPEARANCE_SIZE_T, 142, 160, 39, 9
     EDITTEXT IDC_ADVAPPEARANCE_SIZE_E, 142, 170, 38, 13, ES_RIGHT | WS_GROUP
     CONTROL "", IDC_ADVAPPEARANCE_SIZE_UD, UPDOWN_CLASS, UDS_AUTOBUDDY | WS_BORDER | WS_GROUP |
             UDS_ALIGNRIGHT | UDS_ARROWKEYS | UDS_SETBUDDYINT | WS_CHILD | WS_VISIBLE, 172, 170, 10, 13
-    LTEXT "Couleur 1 :", IDC_ADVAPPEARANCE_COLOR1_T, 180, 160, 34, 9
+    LTEXT "Coule&ur 1 :", IDC_ADVAPPEARANCE_COLOR1_T, 180, 160, 34, 9//FIXME: control is too small to hold all the text. For the backport I simply stripped all " :" in this dlg. Imho that would be a good solution also for master. Some other translations do that as well.
```

Comment:
No, please keep the `:`, this is how it has to be done per Windows UI guidelines (see link I pasted in Mattermost last week or so).
The label needs to be enlarged from 34 to something like 40:

![image](https://github.com/user-attachments/assets/a0df6d20-dcba-4299-b144-cc99aa8c8fe1)

Reply:
I did **keep** it in master. But it must be fixed there to actually fit into the control. This is something native speakers can play with later. That is why I added the FIXME. Resolved. I did not change anything in that regard in master, to avoid having to discuss with you. So you can solve it in the future to match your presonal preference.